aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -100,7 +100,17 @@ Tensor& mul_(Tensor& self, const Tensor& other) {
   return native::mul_out(self, self, other);
 }
 
+// Basic checking for all sub functions.
+static inline void sub_check(const Tensor& self, const Tensor& other) {
+  TORCH_CHECK(self.scalar_type() != kBool || other.scalar_type() != kBool,
```

Comment:
I suggested slightly different wording on the old PR, can you move that over here?

Reply:
Oops, just saw that. Let me change here.